### ISSUE TITLE
Toggle Buttons

### ISSUE THREAD

**Issue by scottaohara** [scottaohara/a11y_styled_form_controls#31]
With `role="switch"` having sporadic support, an alternate "switch" (or different styled) component example utilizing `button aria-pressed="true/false"` would be nice to have.

Note: `input type="checkbox" role="button" aria-pressed="true/false"` has some weird issues that should be reported to the appropriate screen readers/browsers.  Do not use this in new component.
